pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -73,20 +77,26 @@ def __init__(
         monitor: str = 'early_stop_on',
         min_delta: float = 0.0,
         patience: int = 3,
+        max_time: Optional[float] = None,
+        threshold: Optional[float] = None,
         verbose: bool = False,
         mode: str = 'min',
         strict: bool = True,
     ):
         super().__init__()
         self.monitor = monitor
         self.patience = patience
+        self.max_time = max_time
+        self.threshold = threshold
         self.verbose = verbose
         self.strict = strict
         self.min_delta = min_delta
         self.wait_count = 0
         self.stopped_epoch = 0
         self.mode = mode
         self.warned_result_obj = False
+        self.last_time = time()
+        self.elapsed_time = 0.0
```

Comment:
```suggestion
        self._last_time = time()
        self._elapsed_time = 0.0
```
this shall be at least protected...